src/main/java/be/polyscripts/contactmanagerapp/model/Contact.java

```diff
@@ -0,0 +1,47 @@
+package be.polyscripts.contactmanagerapp.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Builder
+@AllArgsConstructor @NoArgsConstructor @Getter @Setter
+@Entity
+@Table(name = "contact")
+public class Contact {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)

```

Comment:
use IDENTITY instead of auto, in some case and for some SGBDs Auto will cause some issues

Reply:
done !